src/App.css

```diff
@@ -421,6 +421,8 @@ input[type='radio'][disabled] {
 /*fix: select makes container to grow*/
 select {
   width: 100%;
+  appearance: auto !important;
```

Comment:
this break admin view

![image](https://user-images.githubusercontent.com/4190728/100749142-8b720e80-33e4-11eb-8eb1-2921adf251e9.png)

Reply:
avant : 

![image](https://user-images.githubusercontent.com/4190728/100755028-0854b680-33ec-11eb-9fde-f6513aa061af.png)

après : 

![image](https://user-images.githubusercontent.com/4190728/100755074-1276b500-33ec-11eb-8d92-13d8aaa37b2a.png)